Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -390,6 +390,11 @@ namespace Camera
         UpdateCamera();
     }
 
+    void CameraComponentController::SetStereoscopicView([[maybe_unused]] const AZ::Quaternion& viewQuat, [[maybe_unused]] uint32_t xrViewIndex)
+    {
+        // todo:: Needs implementation
```

Comment:
maybe add an assert here

            AZ_Assert(false, "Not implemented, xxx");